clustering-map/src/App.js

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import {
+  client,
+  useConfig,
+  useElementColumns,
+  useElementData,
+} from "@sigmacomputing/plugin";
+
+import './App.css';
+import Map from './components/Map';
+
+client.config.configureEditorPanel([
+  { name: "source", type: "element" },
+  { name: "Latitude", type: "column", source: "source", allowMultiple: false, allowedTypes: ['number', 'integer'] },
+  { name: "Longitude", type: "column", source: "source", allowMultiple: false, allowedTypes: ['number', 'integer'] },
+  { name: "Fields", type: "column", source: "source", allowMultiple: true },
+  { name: "Cluster scale", type: "text", placeholder: '1 to 100' },
```

Comment:
you can name the config `clusterScale` or something code-friendlier, and use the `label` property for display.
```suggestion
  { name: "cluster-scale", label: "Cluster scale", type: "text", placeholder: '1 to 100' },
```